docs/.vuepress/components/RandomAppList.vue

```diff
@@ -0,0 +1,40 @@
+<template>
+	<RandomList :n="4", :items="items"></RandomList>
```

Comment:
Please remove the comma between props here. It's causing this console error when navigating to the /use page:
```
DOMException: Failed to execute 'setAttribute' on 'Element': ',' is not a valid attribute name.
```

Reply:
Okay, done!